api/v1/model_handlers.go

```diff
@@ -267,11 +388,11 @@ func (a *Api) freeContainerMetrics(request *restful.Request, response *restful.R
 	response.WriteEntity(exportTimeseries(timeseries, new_stamp))
 }
 
-// parseRequestStartParam obtains the time.Time to be used as the start time during a model Get method.
-// parseRequestStartParam receives a request and a response as inputs, and returns the parsed time.
-func parseRequestStartParam(request *restful.Request, response *restful.Response) time.Time {
+// parseRequestParam parses a time.Time from a named QueryParam.
+// parseRequestParam receives a request and a response as inputs, and returns the parsed time.
+func parseRequestParam(param string, request *restful.Request, response *restful.Response) time.Time {
```

Comment:
nit: shouldn't extra arguments be at the end? Also, looks like its doc needs updating.